docs/api-usage.rst

```diff
@@ -47,6 +48,13 @@ configuration files.
    If the GitLab server you are using redirects requests from http to https,
    make sure to use the ``https://`` protocol in the URL definition.
 
+.. note::
+
+   It is highly recommended to use the final destination in the ``url`` field.
+   What this means is that you should NOT use a URL which redirects as it will
```

Comment:
Sorry for the nitpick - we already have an admonition here so I think this works already 😁 
```suggestion
   What this means is that you should not use a URL which redirects as it will
```

Reply:
Done